### ISSUE TITLE
Clarify requirements for basic usage

### ISSUE THREAD

**Issue by carolineferraby** [statamic/collaboration#100]
Tried adding the Collaboration addon (version 0.8.1) to my Statamic project (version 4.5.7.0) on Laravel 10, and the listed addon "features" (presence indicators for multiple people, fields getting locked, etc.), are not working on the site.

I followed every step in the installation documentation, so I am now at a loss why it isn't working as expected. Wondering if there are any additional requirements or "gotchas" that aren't mentioned in the documenation. Here a few guesses I've brainstormed:

1. When creating a Pusher account, you could create either a "Channels" app or a "Beams" app. I created a Channels app, as that seemed the most appropriate, but maybe I was wrong. 
2. Our site has three different environments (1 domain for each environment), so when initializing our Pusher app, I selected the checkbox to create apps for multiple environments. When adding the app keys to our .env files, I made sure the -development app keys matched the developement .env file and so on, but maybe the addon wasn't intended to be used this way. All of our environments are using Statamic Pro.
3. Instead of flat files, we are using a database. We have also created symbolic links in config/filesystems.php for content files.
4. Users do not have an avatar image field, not sure if that is required.

It would also be helpful to have a way to test the addon. I can view my Pusher app (for the development env, where I am testing) and see graphs that show multiple people/users connecting, however this does not have any affect on the control panel when multiple users are editing the same collection entry.

**Comment by duncanmcclean** [comment]
Have you enabled "Client Events" in your Pusher app's settings?

![CleanShot 2024-04-22 at 20 13 43](https://github.com/statamic/collaboration/assets/19637309/36253f90-9fd7-4def-9ce4-a830a2a8c765)

**Comment by carolineferraby** [comment]
Yes I have enabled client events

**Comment by carolineferraby** [comment]
I logged in as two separate users (using different emails / user accounts) in two browser windows.

**Comment by duncanmcclean** [comment]
> Have you enabled "Client Events" in your Pusher app's settings?

And just to double check, you enabled this in the settings of each of the apps Pusher created? (it creates a different "app" for each environment)

**Comment by carolineferraby** [comment]
Yes correct, I enabled it for each environment (so three times). Though for testing, I've only pushed up the code and tested on the development environment so far.

**Comment by carolineferraby** [comment]
I'm not sure if this is related, but I'll mention it in case it is -- I also ran into the issue where the 'php artisan vendor:publish --tag=collaboration' command did not publish a config file. Instead I got this as the terminal output message: "Publishing [collaboration] assets. Copying directory [vendor/statamic/collaboration/resources/dist/build] to [public/vendor/collaboration/build]."

**Comment by duncanmcclean** [comment]
What does your `composer.json` file look like?

**Comment by carolineferraby** [comment]
[composer.json](https://github.com/statamic/collaboration/files/15068596/composer.json)

**Comment by carolineferraby** [comment]
Okay, this is very strange, but the addon suddenly started working this morning, and I'm not sure why. Not sure if it needed a few days? The code was definitely deployed (on April 19th) because we could see the addon listed in the control panel, and we could see our stats/connections on the pusher app, it just wasn't working in the control panel. I don't think it was a caching issue -- we ran the commands to clear any statamic or laravel cache, and we don't have any caching on our dev server. The only other variable is an issue on dev we were having simultaneously, which is that we ran a migration to copy a bunch of files and accidentally filled the disk so fast that it crashed (now resolved).

When I push up the addon code to our QA site, and I edit the .env file on the QA server, I'll see if I can replicate the issue. I'll see if it works right away, or if I have to give it a few days.
